features/org.wso2.carbon.appmgt.store.feature/src/main/resources/store/extensions/assets/webapp/themes/store/js/logic/businessOwners/businessOwners.js

```diff
@@ -0,0 +1,54 @@
+/*
+ *  Copyright (c) 2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+function merge(businessOwner){
+    var keys = businessOwner.keys;
+    var values = businessOwner.values;
+
+    if(keys != null || keys !="") {
+        var keySet = keys.split("/");
```

Comment:
Use the data structure to store owner details. Don't use string manipulation to populate owner details